tests/format/css/url/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -11,11 +11,47 @@ div {
   background: -fb-url(/images/bg.png);
 }
 
+@font-face {
+  src: url(RobotoFlex-VariableFont_GRAD,XTRA,YOPQ,YTAS,YTDE,YTFI,YTLC,YTUC,opsz,slnt,wdth,wght.ttf);
+}
+
+@font-face {
+  src: url(foo.ttf?query=foo,bar,);
+  src: url(foo.woff2?foo=rgb\\(255,255,0\\));
+}
+
+a {
+  content: url(https://example.com/\\)\\).jpg);
+  content: url(https://example.com/\\(\\(.jpg);
+  content: url(https://example.com/\\ \\ .jpg);
+  content: url(   https://example.com/\\)\\).jpg   );
+  content: url(   https://example.com/\\(\\(.jpg   );
+  content: url(   https://example.com/\\ \\ .jpg   );
+}
+
 =====================================output=====================================
 div {
   background: url(/images/bg.png);
   background: -fb-url(/images/bg.png);
 }
 
+@font-face {
+  src: url(RobotoFlex-VariableFont_GRAD,XTRA,YOPQ,YTAS,YTDE,YTFI,YTLC,YTUC,opsz,slnt,wdth,wght.ttf);
+}
+
+@font-face {
+  src: url(foo.ttf?query=foo,bar,);
+  src: url(foo.woff2?foo=rgb\\(255,255,0\\));
+}
+
+a {
+  content: url(https://example.com/\\)\\).jpg);
+  content: url(https://example.com/\\(\\(.jpg);
+  content: url(https://example.com/\\ \\ .jpg);
+  content: url(   https://example.com/\\)\\).jpg   );
+  content: url(   https://example.com/\\(\\(.jpg   );
+  content: url(https://example.com/\\ \\ .jpg);
```

Comment:
Should we trim here?

Reply:
As I tested with in chrome

```
<style>
body {
  background: url(              1.png?_=      )
}
</style>
```

and

```
<style>
body {
  background: url(              1.png      )
}
</style>
```

They all requesting urls without space.

Reply:
postcss-values-parser throws.
I think it should be a bug.
I'll try newer versions.
```js
const valueParser = require("postcss-values-parser");
valueParser('url(   https://example.com/\\(\\(.jpg   );', {loose: true}).parse()
```

```
Uncaught ParserError: Expected closing parenthesis at line: 1, column 4
    at Parser.error (~/Projects/oss/prettier/node_modules/postcss-values-parser/lib/parser.js:127:11)
    at Parser.parenOpen (~/Projects/oss/prettier/node_modules/postcss-values-parser/lib/parser.js:279:12)
    at Parser.parseTokens (~/Projects/oss/prettier/node_modules/postcss-values-parser/lib/parser.js:235:14)
    at Parser.loop (~/Projects/oss/prettier/node_modules/postcss-values-parser/lib/parser.js:132:12)
    at Parser.parse (~/Projects/oss/prettier/node_modules/postcss-values-parser/lib/parser.js:51:17)
```

Reply:
v3.2.1 seems good.
```js
const { parse } = require('postcss-values-parser');
parse('url(   https://example.com/\\(\\(.jpg   )');
```

<details>

```
<ref *1> Root {
  raws: { semicolon: false, after: '' },
  type: 'root',
  nodes: [
    Func {
      raws: [Object],
      name: 'url',
      type: 'func',
      isColor: false,
      isVar: false,
      nodes: [Array],
      parent: [Circular *1],
      source: [Object],
      params: '(   https://example.com/\\(\\(.jpg   )'
    }
  ],
  source: {
    input: Input {
      css: 'url(   https://example.com/\\(\\(.jpg   )',
      hasBOM: false,
      id: '<input css 1>'
    },
    start: { line: 1, column: 1 }
  },
  toString: [Function: bound toString]
}
```

</details>

But v3.0.0 is big breaking release. https://github.com/shellscape/postcss-values-parser/releases
Which way do you like?
- Bump postcss-values-parser  on next branch via another PR first, and then work on this bug.
- Leave `'url(   https://example.com/\\(\\(.jpg   )'` and merge this PR first. And then try to bump postcss-values-parser on next branch. Finally, tackle `'url(   https://example.com/\\(\\(.jpg   )'` (or, perhaps it will be resolved by update).

Reply:
> And then try to bump postcss-values-parser

Too hard..

Reply:
Will it conflict with your work if I'll try to bump postcss-value-parser?
Referring following, I'm asking you.
> I want to make some change to the value-comma_group part

https://github.com/prettier/prettier/pull/14480

Reply:
No. Go a head, but it won't be easy, I've tried few times since 2 years ago.

Reply:
Last attempt by other maintainer. https://github.com/prettier/prettier/pull/11667

Reply:
Thank you for your replies.
Sounds so hard!
Any way, I'll try it, though I will be likely to give up 🔥

Reply:
@fisker 
Hi, I have tried migrating postcss-values parser from v2 to v6.
Eventually, I gave up because https://github.com/shellscape/postcss-values-parser/issues/140 prevents us from migrating.
May I summarize what I investigated for migration somewhere (perhaps an issue or a discussion)?